assets/js/modules/search-console/datastore/index.js

```diff
@@ -29,6 +29,7 @@ const baseModuleStore = Modules.createModuleStore( 'search-console', {
 	settingSlugs: [
 		'propertyID',
 	],
+	adminPage: 'googlesitekit-module-search-console',
```

Comment:
This should also receive `requiresSetup: false`, since Search Console does not have any setup in the plugin.

Reply:
Added with default to false